src/style.css

```diff
@@ -1 +1,47 @@
-/* add styles here */
+body {
+  margin: 0;
```

Comment:
Using the `body` tag to apply styles is against the checklist. It's better to use a class to apply styles. (CSS Checklist: Don't use tag names for styling except `html` and `body`)